Update 20.treatments.md (#309)
greenelab/covid19-review: content/20.treatments.md

The study retrospectively investigated the effects of HCQ (n=97), HCQ+AZ (Azithromycin) (n=113) and no HCQ (n=158) on 368 patients for the primary outcomes of death and the need for mechanical ventilation. 
Standard supportive care was rendered to all patients.
The study included only men, with a median age of 65 years.
Women were excluded due to their considerably lower representation in the available data (N=17). 
The rate of death in the HCQ-only treatment condition was 27.8%, compared to 22.1% in the HCQ+AZ and 14.1% in the no-HCQ cohort.
The risk of death was higher in the HCQ-only group compared to the no-HCQ group (adjusted hazard ratio: 2.61, p<0.03).
Further, the risk of ventilation was similar across all three groups: (adjusted hazard ratio: 1.43 (HCQ) and 0.43 (HCQ+AZ) compared to no HCQ).
The study thus showed evidence of an association between increased mortality and HCQ in this cohort of COVID-19 patients.
Also, the HCQ and HCQ+AZ conditions were not associated with any advantage in rates of mechanical ventilation. 

> The study had a few limitations.
> The study was not randomized and did not include placebo.
> Also, baseline vital signs, laboratory tests, and prescription drug use were significantly different among the three groups, and all of these factors could potentially influence treatment outcome. 

Reviewer: ```suggestion
The study had a few limitations: it was not randomized, and the baseline vital signs, laboratory tests, and prescription drug use were significantly different among the three groups.
All of these factors could potentially influence treatment outcome. 
```

Overall, the study provides promising data, although all of the findings still need to be validated in independent population cohorts.
Their safety analysis indicated that the adverse effects of administering HCQ to mild and moderate COVID-19 cases were manageable.
Further investigation is necessary to confirm whether the drug indeed ameliorates symptoms and reduces inflammatory response.

> A retrospective analysis [@doi:10.1101/2020.04.16.20065920] was conducted on the data from 368 SARS-CoV-2 patients in all United States Veteran Health Administration medical centers.

Reviewer: ```suggestion
A retrospective analysis [@doi:10.1101/2020.04.16.20065920] was conducted on the data from 368 SARS-CoV-2 patients across all United States Veteran Health Administration medical centers.
```

> The study investigated the effects of HCQ (n=97), HCQ+AZ (Azithromycin) (n=113) and no HCQ (n=158) in 368 patients retrospectively for primary outcome of death and need for mechanical ventilation. 

Reviewer: ```suggestion
The study retrospectively investigated the effects of HCQ (n=97), HCQ+AZ (Azithromycin) (n=113) and no HCQ (n=158) on 368 patients for the primary outcomes of death and the need for mechanical ventilation. 
```

Standard supportive care was rendered to all patients.

> The study included only men of median age 65 years while women were excluded due to considerably less in number (17). 

Reviewer: ```suggestion
The study included only men, with a median age of 65 years.
Women were excluded due to their considerably lower representation in the available data (N=17). 
```

> Deaths in HCQ only was 27.8%, 22.1% in HCQ+AZ and 14.1% in no HCQ cohort.

Reviewer: ```suggestion
The rate of death in the HCQ-only treatment condition was 27.8%, compared to 22.1% in the HCQ+AZ and 14.1% in the no-HCQ cohort.
```

> Death risk was higher in HCQ only group compared to no HCQ group (adjusted hazard ratio, 2.61, p<0.03).

Reviewer: ```suggestion
The risk of death was higher in the HCQ-only group compared to the no-HCQ group (adjusted hazard ratio: 2.61, p<0.03).
```

> Further, the risk of ventilation was similar in all three groups: (adjusted hazard ratio, 1.43 (HCQ) and 0.43 (HCQ+AZ) compared to HCQ). The study showed evidence of increased mortality on the use of HCQ in this cohort of SARS-CoV-2.

Reviewer: ```suggestion
Further, the risk of ventilation was similar across all three groups: (adjusted hazard ratio: 1.43 (HCQ) and 0.43 (HCQ+AZ) compared to no HCQ).
The study thus showed evidence of an association between increased mortality and HCQ in this cohort of COVID-19 patients.
```

> Also, no advantage was rendered upon the use of HCQ and HCQ+AZ towards mechanical ventilation. 

Reviewer: ```suggestion
Also, the HCQ and HCQ+AZ conditions were not associated with any advantage in rates of mechanical ventilation. 
```

> The study had few limitations.

Reviewer 2: ```suggestion
The study had a few limitations.
```

> The study had few limitations.

Reviewer 3: The study had a few limitations, including that it was not a randomized trial and did not include a placebo group.

> The study was not randomized control and did not include placebo.

Reviewer 2: ```suggestion
The study was not randomized and did not include placebo.
```
Is the point of noting a lack of placebo here to say that the difference was HCQ / HCQ+AZ / nothing? I'd usually be most concerned about not having a placebo if we saw a positive effect in both drug arms but not in the "nothing" setting. Is there evidence that a placebo can have negative effects as well as beneficial ones? Definitely the lack of randomization makes it hard to get into causality.

Author: @cgreene : I think you are right. Since the result is negative in HCQ and HCQ+AZ, the point of having placebo is not required. We can strike out the word" placebo".

> The study was not randomized control and did not include placebo.

Reviewer 3: Remove sentence, since it is combined with sentence above.

> Also, the baseline vital signs, laboratory tests, prescription drug use were significantly different in three groups which can potentially have an effect on treatment outcome. 

Reviewer: ```suggestion
Also, baseline vital signs, laboratory tests, and prescription drug use were significantly different among the three groups, and all of these factors could potentially influence treatment outcome. 
```

> Also, the baseline vital signs, laboratory tests, prescription drug use were significantly different in three groups which can potentially have an effect on treatment outcome. 

Reviewer 3: What laboratory tests were different?

> Further the authors acknowledge that the effect of the drugs might be different in females as well as in pediatric subjects as these subjects were not the part of the study.

Reviewer 3: Furthermore, the authors acknowledge that the effect of the drugs might be different in females and pediatric subjects, since these subjects were not the part of the study.

> ##### Summary

Reviewer 2: ```suggestion

##### Summary
```

Standard supportive care was rendered to all patients.
The study included only men, with a median age of 65 years.
Women were excluded due to their considerably lower representation in the available data (N=17). 
The rate of death in the HCQ-only treatment condition was 27.8%, compared to 22.1% in the HCQ+AZ and 14.1% in the no-HCQ cohort.
The risk of death was higher in the HCQ-only group compared to the no-HCQ group (adjusted hazard ratio: 2.61, p<0.03).
Further, the risk of ventilation was similar across all three groups: (adjusted hazard ratio: 1.43 (HCQ) and 0.43 (HCQ+AZ) compared to no HCQ).
The study thus showed evidence of an association between increased mortality and HCQ in this cohort of COVID-19 patients.
Also, the HCQ and HCQ+AZ conditions were not associated with any advantage in rates of mechanical ventilation. 
The study had a few limitations.
The study was not randomized and did not include placebo.
Also, baseline vital signs, laboratory tests, and prescription drug use were significantly different among the three groups, and all of these factors could potentially influence treatment outcome. 

> Further the authors acknowledge that the effect of the drugs might be different in females as well as in pediatric subjects as these subjects were not the part of the study.

Reviewer: ```suggestion
Furthermore, the authors acknowledge that the effect of the drugs might be different in females and pediatric subjects, since these subjects were not the part of the study.
```
